genesis-creator/examples/genesis6.toml

```diff
@@ -3,7 +3,7 @@
 # - 5 bakers
 # - one additional foundation account
 # - genesis time set to the time when the tool runs
-# - average block time set to 5s
+# - average block time set to 1s
```

Comment:
This is also an incorrect comment for P6. It should talk about minimum time I assuem.